.github/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: node_js CI
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
nit: From the [previous comment](https://github.com/edx/frontend-logging/pull/20#discussion_r1594034081), I would still recommend only running this for `pull_request`. There's not really a need to run this workflow for commits to `master`.